launch/load_vrt_launch.xml

```diff
@@ -0,0 +1,19 @@
+<launch>
+    <arg name="rviz" default="false"/>
+
+    <!-- Default CMAC https://maps.app.goo.gl/NXJ9JC23oskMDaEx8-->
+    <arg name="map_origin_latitude" default="-35.363266"/>
+    <arg name="map_origin_longitude" default="149.165199"/>
+
+    <node pkg="tf2_ros" exec="static_transform_publisher" name="world_map" args="--frame-id world --child-frame-id map"/>
+
+    <node pkg="grid_map_geo" exec="map_publisher" name="map_publisher" output="screen"> 
+	    <param name="gdal_dataset_path" value="/vsizip/$(find-pkg-share grid_map_geo)/resources/ap_srtm1.vrt.zip"/>
```

Comment:
Does this mean that we need to store the whole VRT file locally?

Reply:
Nope; I should switch this over to use the VRT hosted online and remove this from the repo. Thanks for the reminder.

Reply:
c08463048f8915f76bdc9e78fd75bfee7e950268

Deleted it and switched over the web hosted VRT now.